### ISSUE TITLE
Markdown monospace support

### ISSUE THREAD

**Issue by zetok** [qTox/qTox#2999]
“It would be very cool if qTox could support code highlighting, or at least monospace and indentation. So, +1.”

by @xenohunter in https://github.com/tux3/qTox/issues/2497#issuecomment-164678436

e.g. some `monospace` markdown

**Comment by sudden6** [comment]
wasn't there a "code" formatting already planned?

**Comment by zetok** [comment]
@sudden6 

Kinda.

This issue is specifically for tracking just monospace support, since this is currently the only mode that can work without problems, considering that message splitting can mess up code blocks, and some sort of plan for handling that case should be in place for code blocks to be implemented.

I.e. for codeblocks, there should be opened a new issue, and there plan for handling things could be discussed.

tl;dr: planned - yep; concise and ready plan doesn't exist yet though.

**Comment by ProMcTagonist** [comment]
How does message splitting affect code blocks in a way it doesn't affect other markup?

**Comment by zetok** [comment]
> How does message splitting affect code blocks in a way it doesn't affect other markup?

Affects to higher degree, which makes it ~unacceptable. I.e. codeblock is way more likely to have thousands of characters which would trigger the message split than just a normal piece of text with some basic markdown.

All those problems stem from toxcore not doing the message splitting :/
